lpa-submissions-e2e-tests/cypress/plugins/index.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+module.exports = (on, config) => {
+  // `on` is used to hook into various events Cypress emits
+  // `config` is the resolved Cypress config
+}
+
+
+const cucumber = require('cypress-cucumber-preprocessor').default
+
+module.exports = (on,config) =>{
+  on('file:preprocessor',cucumber())
+}
+
+module.exports = (on, config) => {
+  on('task', {
+    log (message) {
+      console.log(message)
+
+      return null
+    }
+  })
+}
```

Comment:
Add newline at end - use [editorconfig](https://editorconfig.org/) to do this for you